lib/fluent/plugin/in_wmi.rb

```diff
@@ -32,8 +44,14 @@ def start
       timer_execute(:wmi_timer, @interval) do
         wmi = WmiLite::Wmi.new
         time = Fluent::Engine.now
-        wmi.instances_of(@class_name).each do |instance|
-          router.emit(@tag, time, instance.property_map)
+        if @class_name
+          wmi.instances_of(@class_name).each do |instance|
```

Comment:
Can we use `@wmi` instance which is `WmiLite::Wmi` object instead of creation in every time?